src/views/Log.vue

```diff
@@ -574,11 +584,6 @@ export default {
       this.fileDisabled = true
       this.logFiles = []
     },
-    autoScrollToggle () {
-      this.autoScroll = false
-      const index = this.controlGroups[0].controls.findIndex(obj => obj.key === 'autoScroll')
-      this.controlGroups[0].controls[index].value = this.autoScroll
-    }
```

Comment:
By moving the `controlGroups` to a computed property, we can avoid the need to manually do this